compose.yaml

```diff
@@ -5,6 +5,6 @@ services:
       environment:
         - POSTGRES_DB=${POSTGRES_DB}
         - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-        - POSTGRES_USER$=${POSTGRES_PASSWORD}
+        - POSTGRES_USER=${POSTGRES_USER}
       ports:
-        - ${POSTGRES_PORTS}
+        - ${POSTGRES_PORT}:${POSTGRES_PORT}
```

Comment:
nit: I'd probably squash e81d6eff3e1b51542c1c9f35ac516970b72baa9e with 628b513698fe4d5fc60805218469946a1b30746d, because these changes can not go without each other, but it is not necessary